monitoring/map.py

```diff
@@ -16,16 +17,17 @@ class Map(Task):
     """Map Processing Task"""
 
     def run(self, context):
+        del context
```

Comment:
I just wanted to be clear with the fact that the context is not used but I forgot how to do this properly in Python :) fixed this [by replacing `context` with `_` in the function declaration](https://www.geeksforgeeks.org/unused-local-variable-in-python/)

Reply:
Okay, now I understand your intention. However, every `Task` should have a `run()` function with the signature `run(self, context)` (see [here](https://github.com/uwefladrich/scriptengine/blob/8ddb6284a593fd3e6e8ec09d66b4decbf7f49b86/scriptengine/tasks/core/task.py#L96-L99)), i.e. exactly one positional argument besides `self` and since already the base `Task` class calls this argument `context`, it is safe to assume that this argument should be the context. So I think it would be rather clear if you used the same signature, even if the context is not used.

Reply:
Thinking a bit more about this I realise that you do not have to implement a `run()` function at all here. Because if all you want to achieve is that one cannot call the function, then this is the behaviour already inherited from the base class! Maybe I should adapt the error message in `Task.run()` in order to write the actual class name.

Reply:
The latter seems like a clearer way of dealing with this. I'll try it out :)

Reply:
fixed in 10a5b70